src/App.js

```diff
@@ -25,7 +25,7 @@ class App {
 
     const authenticationChanged = this.isAuthenticated !== isAuthenticated;
     const userLoggedOut = authenticationChanged && !isAuthenticated;
-    const userLoggedIn = authenticationChanged && !user.isEmpty();
+    const userLoggedIn = authenticationChanged && Object.keys(user).length > 0;
```

Comment:
is it possible for `user` to be `undefined`? If yes, this will fail, otherwise looks good 👍

Reply:
I used this:
```
export const isEmpty = (obj) => {
  for(var key in obj) {
      if(obj.hasOwnProperty(key))
          return false;
  }
  return true;
};
```

Reply:
Thanks for the review :+1:!. 
It seems it's always defined, otherwise, `user.isEmpty()` would've also failed. 
I think that the initial state of the session reducer from the session library has an empty object for the user.

Reply:
Since we are using lodash, why not using lodash's `isEmpty`?

Reply:
I didn't know the project had lodash! Thanks!